PINCache/PINDiskCache.m

```diff
@@ -720,15 +722,11 @@ - (id)objectForKeyedSubscript:(NSString *)key
         return nil;
     
     id <NSCoding> object = nil;
-    NSURL *fileURL = nil;
+    NSURL *fileURL = [self encodedFileURLForKey:key];
     
     [self lock];
-        fileURL = [self _locked_encodedFileURLForKey:key];
-        object = nil;
-        
-        if ([[NSFileManager defaultManager] fileExistsAtPath:[fileURL path]] &&
+        if (!self->_ttlCache || self->_ageLimit <= 0 || fabs([[_dates objectForKey:key] timeIntervalSinceDate:now]) < self->_ageLimit) {
```

Comment:
Yes because initWithContentsOfFile returns nil if the file doesn't exist / doesn't throw an exception.